### ISSUE TITLE
Confirmation of v1 glyph set

### ISSUE THREAD

**Issue by davelab6** [TypeNetwork/Roboto-Flex#239]
@EbenSorkin asked me about the glyph set in comparison to "regular" Roboto. This is a great question and worth explaining somewhere, so I'll start here :) 

The "regular" Roboto is somewhat ambiguous... to recap, there's the "Roboto Static" TTF files that today (but this will soon change) are available from GF APIs for web and android, and last released by @m4rc1e at https://github.com/googlefonts/roboto/releases circa 2017. And then there's the "Roboto Classic" VF, which is those styles (plus 2 new min/max condensed named styles) as `wght` and `wdth` axes (plus 2 new min/max condensed named styles, so the design space is rectangular), that FB/TN produced and @m4rc1e released at https://github.com/typenetwork/Roboto/releases circa 2020. I'd prefer any comparisons of Roboto Flex to be made to the latter (Roboto Classic), since that is soon about to become "the" Roboto on GF APIs - and likely the final version ever, modulo any bug fixes. 

But importantly for this topic, the glyph set for both static and classic versions should be identical :)

With that out of the way, as to what the expected glyph set for Flex is, it is a small fraction of what is in Roboto Classic, which as I recall is some ~3,500 in total, and predates the Noto project. 

I believe the big obvious glyph set "missing" from Roboto Flex is small caps, across all 3 scripts; but my hope is that the parametric axes plus font-size can be used to create high quality small caps without bulking the file size. AFAIK that isn't something that David has demonstrated yet. @dberlow, what do you think about adding that to https://variablefonts.typenetwork.com or a future publication? 

Finally, the glyph set is likely related to the Google Latin Pro, Google Cyrillic Pro, and Google Greek Plus sets defined in  https://github.com/googlefonts/glyphsets/tree/main/Lib/glyphsets/encodings/GF%20Glyph%20Sets but I believe @dberlow had refined the exact glyph set in some way, so there may be differences which we could discuss.

**Comment by dberlow** [comment]
There are three kind of glyph sets we need to clarify. There is a **build set**, which includes all the glyphs required to build all those that appear in response to character input, which makes a second **glyph set** . And third are the glyphs that appear in response to feature requests that are encountered, a **glyphs and features set**.

To this, during this project, was added a fourth kind, which are polytonic glyphs handled by Harfbuzz that are extensive, take up a lot of space and do not need to be assembled in the font, we were told, as I recall. 

This discussion also includes to add a fifth set, a glyph-to-character/feature route via design space deflection, i.e. the definition of glyph locations other than the current location, where other features of the current location are found instead of other sizes or styles normally found around the designspace. . 

Bottom line, I'm not sure static and classic are identical, but I know the spec for Extremo/Delta/Flex was compatible when Harfbuzz is present. In addition, the build set of Flex has changed in the process of file size reduction, but whatever the proper comparison is needed, let us know, I have the Classic Regular Dave mentions above, and Flex has a Regular too.

**Comment by davelab6** [comment]
What's possible with features, and what's possible with Harfbuzz, should be the same. No?
